data_dir/nemesis.yml

```diff
@@ -97,6 +104,7 @@
   - disruptive = False
   - kubernetes = True
   - limited = True
+  - schema_changes = True
```

Comment:
@roydahan you test doesn't cover the tags/labels of nemesis, hence changes in tags doesn't force update to those files

Reply:
I didn't think about that...

Reply:
I would add it once adding the 2nd part of running from the nemesis file